public/app/percona/inventory/Inventory.service.ts

```diff
@@ -54,12 +52,13 @@ export const InventoryService = {
     return result;
   },
   getNodes(body = {}, token?: CancelToken) {
-    return api.post<NodeListDBPayload, object>(`/v1/management/Node/List`, body, false, token);
+    // todo: address body
```

Comment:
Please take a look at handling of the `getNodes` body.

Reply:
Yes, no address in the list of request parameters.